### PR TITLE
Feat: Image-Card component

### PR DESCRIPTION
**Pull request by erinfm**
Adds component for Image Cards, and image grids on pages using image cards.

Deployed preview: https://deploy-preview-24--satellytes-website-new.netlify.app/

**Code changes + additions:**

-  Image-Card component created. This component takes an image (using GraphQL and Gatsby Image), and an optional title and subtitle. The image will be resized to a 1/1 aspect ratio.
- Added internal and external link components.
- Added image grids and images/image cards to Services, About and Clients pages.

**Screenshots:**

<details>
<summary>Services Page Desktop</summary>

Screenshot 2020-06-23 at 21 16 53
</details>

<details>
<summary>Services Page Mobile</summary>

Screenshot 2020-06-23 at 21 17 11
</details>

<details>
<summary>About Page Desktop</summary>

Screenshot 2020-06-23 at 21 20 34
</details>

<details>
<summary>About Page Mobile</summary>

Screenshot 2020-06-23 at 21 21 10
</details>

<details>
<summary>Clients Page Desktop</summary>

Screenshot 2020-06-23 at 21 15 54
</details>

<details>
<summary>Clients Page Mobile</summary>

Screenshot 2020-06-23 at 21 16 30
</details>

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -77,11 +85,90 @@ const AboutPage: React.FC = () => {
         </GridItem>
         <GridItem xs={12} md={8}>
           <AboutSubTitle>Team</AboutSubTitle>
-          <div>TODO: TeamCards</div>
         </GridItem>
       </Grid>
+      <ImageCardGrid>
+        <GridItem xs={6} sm={3}>
```
Comment (feedm3) [inline src/pages/about.tsx]: On the samllest `sm`, the images are quite small. I would therefore go for `sm=4` and `md=3`.

File: src/components/links/internal-link.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+import { Link } from 'gatsby';
+
+export const InternalLink = styled(Link)`
```
Comment (feedm3) [inline src/components/links/internal-link.tsx]: I would put both links into the same `link.tsx` file. Makes it easier to manage them, as both have a bunch of stuff in common.

File: src/pages/clients.tsx [hunk 3]
```diff
@@ -35,11 +40,71 @@ const ClientsPage: React.FC = () => {
...
+        <GridItem xs={12} sm={4}>
+          <InternalLink to="/">
+            <ImageCard
+              alt="Lewandowski"
+              image={data.imagePlaceholder}
+              largeTitle="Client Two"
+            />
+          </InternalLink>
+        </GridItem>
+        <GridItem xs={12} sm={4}></GridItem>
```
Comment (feedm3) [inline src/pages/clients.tsx]: to make the item invisible it's more clear if you make it `xs=0` instead of `12`

**Review by feedm3 — CHANGES_REQUESTED** [review]

File: src/pages/about.tsx (hunk 1 above)
Comment (miffili) [reply src/pages/about.tsx]: I don't agree. It looks exactly as in the design.
And as long as we not know exactly how the pictures will look like, I would roll with it as it is.

**Review by miffili — COMMENTED** [review]
Since you have image grids in multiple places, would it make sense to refactor them in own components? Just a thought. 🙂  To clean up the code a bit & get the repetitive code out of the page files.

Otherwise fine to merge in my opinion.

File: src/components/links/internal-link.tsx (hunk 2 above)
Comment (miffili) [reply src/components/links/internal-link.tsx]: +1

File: src/pages/clients.tsx (hunk 3 above)
Comment (miffili) [inline src/pages/clients.tsx]: Are these the empty tiles? I would put them in a component. This markup doesn't make it clear, tbh.
```javascript
const EmptyGridItem: React.FC = () => (
  <GridItem xs={0} sm={4} />
);
```
And use it in the specific locations. 😇

**Review by miffili — APPROVED** [review]

File: src/pages/clients.tsx (hunk 3 above)
Comment (erinfm) [reply src/pages/clients.tsx]: how do i then implement two consecutive empty grid items? when i change the code to  
Screenshot 2020-06-25 at 10 58 39
it breaks the layout, there is now only one empty grid item, not two
Screenshot 2020-06-25 at 10 59 19

Comment (feedm3) [reply src/pages/clients.tsx]: Like you did it was already correct 😉 what I meant is: the `xs=12` GridItem is empty. So it doesn't take any space. You can therefore also make it `xs=0`, which hides the whole item. The result will (should) be the same, but the code will be more clear that this item is only there to give some spacing (`sm=4`) on larger devices.

Comment (erinfm) [reply src/pages/clients.tsx]: Aaah right, ok, i get it! Changed it to xs={0} s={4} 👍

File: src/components/image-card/image-card.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  ImageCardTitleLarge,
+  ImageCardTitle,
+  ImageCardSubtitle,
+} from '../typography/typography';
+import { up } from '../breakpoint/breakpoint';
+
+interface ImageCardProps {
+  image: {
+    childImageSharp: {
+      fluid: {
```
Comment (feedm3) [inline src/components/image-card/image-card.tsx]: you can use 

`import { FluidObject } from 'gatsby-image';`

File: src/pages/about.tsx [hunk 5]
```diff
@@ -77,11 +81,39 @@ const AboutPage: React.FC = () => {
...
+        teamMembers={teamMemberData}
+        imagePlaceholder={data.imagePlaceholder}
+      />
     </Layout>
   );
 };
 
 export default AboutPage;
+
+// TODO: where should we put this data?
```
Comment (feedm3) [inline src/pages/about.tsx:96]: Into a json, but let's do this in a separate PR :)

**Review by feedm3 — APPROVED** [review]
Besides one small change everything is good to go for me 👍

**Review by feedm3 — APPROVED** [review]
